packages/dd-trace/src/appsec/remote_config/index.js

```diff
@@ -35,6 +35,7 @@ function enable (config) {
 function enableAsmData (appsecConfig) {
   if (rc && appsecConfig && appsecConfig.rules === undefined) {
     rc.updateCapabilities(RemoteConfigCapabilities.ASM_IP_BLOCKING, true)
+    rc.updateCapabilities(RemoteConfigCapabilities.ASM_USER_BLOCKING, true)
```

Comment:
We should disable the capability in desableAsmData. 
We are starting to do it in this pending to merge PR: https://github.com/DataDog/dd-trace-js/pull/2654